core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -658,6 +665,19 @@ public static boolean defaultUseHttp1Pipelining() {
         return DEFAULT_USE_HTTP1_PIPELINING;
     }
 
+    /**
+     * Returns the default value for http2 PING timeout.
+     * Note that this flag is only in effect when DEFAULT_SERVER_IDLE_TIMEOUT_MILLIS for server and
+     * DEFAULT_DEFAULT_CLIENT_IDLE_TIMEOUT_MILLIS for client are greater than 0
+     *
+     * <p>The default value of this flag is {@value #DEFAULT_DEFAULT_HTTP2_PING_TIMEOUT_NANOS}. Specify the
+     * {@code -Dcom.linecorp.armeria.defaultHttp2PingTimeoutNanos=<integer>} JVM option to override
+     * the default value.
+     */
+    public static long defaultHttp2PingTimeoutNanos() {
+        return DEFAULT_HTTP2_PING_TIMEOUT_NANOS;
```

Comment:
Do we need sub-millisecond precision for ping timeouts? How about using milliseconds, like other idle timeout flags?

Reply:
Sure. Let me change this.